Volontulo/app/src/main/java/com/stxnext/volontulo/model/Offer.java

```diff
@@ -0,0 +1,59 @@
+package com.stxnext.volontulo.model;
+
+import android.support.annotation.DrawableRes;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
```

Comment:
It think it's better to use the same names from offer create form (web: http://volontuloapp.org/offers/create).
But it's only a suggestion - for future ;)